### PR TITLE
Add and update documentation (#8 / #40)

### PR DESCRIPTION
**Pull request by Bukama**
```
This PR adds general documentation and updates some mistakes of the documentation of the issue report.

related: #8 
PR: #40
```

### CODE REVIEW

File: docs/issue-report.adoc [hunk 1]
```diff
@@ -71,9 +71,11 @@ This example shows a report which include:
 Several values are configurable by system properties.
```
Comment (Michael1993) [inline docs/issue-report.adoc:71]: Why system properties? Wouldn't a `testreport.properties` file (in the test resources) make more sense?

File: docs/issue-report.adoc [hunk 2]
```diff
@@ -49,7 +49,7 @@ This example shows a report which include:
```
Comment (Michael1993) [inline docs/issue-report.adoc:49]: A bit above: `planed` -> `planned`

File: docs/project-page.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+---
+:page-layout: single
+:page-permalink: /test-reports/
+:page-header: { overlay_image: {site-images}/test-reports-full.jpg, title_alignment: left }
+:page-title: Test Reports
+
+== Motivation
+
+The `Test Reports` provides additional information in (new) reports of the test results.
+The project was initialized as the current JUnit test report can not publish additional attributes or information, e.g. values created by extensions.
```
Comment (Michael1993) [inline docs/project-page.adoc]: "Initialized"? Just say "started".

File: docs/project-page.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+The library itself does not create one single report, but provides test report provide the user can use.
```
Comment (Michael1993) [inline docs/project-page.adoc]: This line is not clear. Does the library create test reports or not? What is a "test report provider"?

Also, I think you meant to write "but provides test report provide**r(s)**"?

File: docs/issue-report.adoc (hunk 1 above)
Comment (Bukama) [reply docs/issue-report.adoc:71]: To be file-independent and be able to use the `SystemPropertyExtension` :)

**Comment by Bukama** [comment]
Thanks @Michael1993 Updated the parts you mentioned

**Pull request merged**